### DISCUSSION TITLE
CSV files for edges & vertices available ?

### DISCUSSION THREAD

**Discussion by JordanYu21** [JacobBumgarner/VesselVio#62] [Q&A] [answered]
Currently working with Vessel Vio for microvascular network visualization, is there sample data including the edges and vertices .csv ?

**Comment by JacobBumgarner** [comment, marked as answer]
Yes - I added some demo CSV files to the main repo. As noted, these data came from VesselGraph.
See here:
https://github.com/JacobBumgarner/VesselVio/tree/main/sample_data/demo_csv
